### PR TITLE
Fixed vocabulary delete API endpoint request

### PR DESCRIPTION
**Pull request by karelklima**
Related to this change in SGoV API: https://github.com/opendata-mvcr/sgov/pull/150

### CODE REVIEW

**Review by Mighantos — APPROVED** [review]
LGTM, maybe unify Iri/Uri naming

File: src/components/vocabularies/DeleteVocabularyForm.tsx [hunk 1]
```diff
@@ -44,7 +44,7 @@ const DeleteVocabularyForm: React.FC<DeleteVocabularyFormProps> = ({
       onSubmit={onSubmit}
     >
       <Hidden name="workspaceId" value={workspace?.id || ""} />
-      <Hidden name="vocabularyId" value={vocabulary?.id || ""} />
+      <Hidden name="vocabularyIri" value={vocabulary?.uri || ""} />
```
Comment (Mighantos) [inline src/components/vocabularies/DeleteVocabularyForm.tsx:47]: here

Comment (karelklima) [reply src/components/vocabularies/DeleteVocabularyForm.tsx:47]: That's not so simple. The discrepancy is a result of historical API of SGoV and chances are that if I were to replace `uri` with `iri` some endpoint will start failing...

Comment (karelklima) [reply src/components/vocabularies/DeleteVocabularyForm.tsx:47]: We should be using IRI everywhere ideally.

**Pull request merged**